packages/contracts/contracts/engine/V3/GenArt721CoreV3_Engine.sol

```diff
@@ -224,7 +223,7 @@ contract GenArt721CoreV3_Engine is
     bool public immutable autoApproveArtistSplitProposals;
 
     /// version & type of this core contract
-    bytes32 constant CORE_VERSION = "v3.1.4";
+    bytes32 constant CORE_VERSION = "v3.2.4";
```

Comment:
nit: should we make this 3.2.0?

Reply:
good call - also updated Flex to be v3.2.1! updated in [aaf33ff](https://github.com/ArtBlocks/artblocks-contracts/pull/1455/commits/aaf33ff2b1542cbfd5c7f7a1b789b4c7932c07ad)